components/restservice/scripts/preconfigure.py

```diff
@@ -1,32 +1,27 @@
 #!/usr/bin/env python
 
-import tempfile
-import os
-from os.path import join, dirname
-
 from cloudify import ctx
 
-ctx.download_resource(
-    join('components', 'utils.py'),
-    join(dirname(__file__), 'utils.py'))
-import utils  # NOQA
-
-REST_SERVICE_NAME = 'restservice'
-
 
 def preconfigure_restservice():
 
-    rest_service_home = '/opt/manager'
+    target_runtime_props = ctx.target.instance.runtime_properties
+    source_runtime_props = ctx.source.instance.runtime_properties
+
+    ctx.logger.info('Reading internal_rest_host from manager_configuration...')
```

Comment:
generally, check in each preconfigure which runtime properties are actually used